apps/daisy/app.js

```diff
@@ -540,14 +686,15 @@ function buzzer(n) {
 // timeout used to update every minute
 var drawTimeout;
 
-// schedule a draw for the next minute
+// schedule a draw for the next minute or every 5 seconds
```

Comment:
Should this be 3? Or perhaps we just say N seconds to avoid the comment becoming stale

```suggestion
// schedule a draw for the next minute or every N seconds
```

Reply:
I'd either call it out to 3, or have the 3second amount be a const in the code. I don't think the 3 seconds will change, but having this info in 3 seperate places makes me consider making is a const.

Reply:
Changed [here](https://github.com/espruino/BangleApps/pull/3759/commits/f431258eafca3167610b295b625fda24fa87ea38).

Reply:
Const is a great idea, thanks!